hatchery/pods.go

```diff
@@ -65,6 +89,57 @@ func getPodClient() corev1.CoreV1Interface {
 	return podClient
 }
 
+// Generate EKS kubeconfig using AWS role
+func NewEKSClientset(userName string /*cluster *eks.Cluster, roleARN string*/) (corev1.CoreV1Interface, error) {
+	pm := Config.PayModelMap[userName]
+	roleARN := "arn:aws:iam::" + pm.AWSAccountId + ":role/csoc_adminvm"
```

Comment:
minor feedback: because the string ":role/csoc_adminvm" is used in multiple places, it might be helpful in the future to pull it from a config variable struct. but for now this is alright, since we're on a deadline ~

Reply:
Yes, eventually this will be pulled from a db and not a config, but I agree.